tests/Oryx.BuildImage.Tests/ExecCommandTest.cs

```diff
@@ -0,0 +1,92 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.Oryx.BuildScriptGenerator.Node;
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.BuildScriptGenerator.Php;
+
+namespace Microsoft.Oryx.BuildImage.Tests
+{
+    public class ExecCommandTest : SampleAppsTestBase
```

Comment:
Add a test where the user provided command exits with a non successful exit code